controllers/reconcilers/prometheus_installation/prometheus_installation_reconciler.go

```diff
@@ -142,42 +159,56 @@ func (r *Reconciler) reconcileCatalogSource(ctx context.Context, cr *v1.Observab
 			Name:      "prometheus-catalogsource",
 			Namespace: cr.GetPrometheusOperatorNamespace(),
 		},
+		Spec: v1alpha1.CatalogSourceSpec{
+			Image: "quay.io/integreatly/custom-prometheus-index:1.0.0",
```

Comment:
I don't think this entry is necessary?

Reply:
You're right it's not necessary. I'll remove this. Thanks